Resources/ArmTemplates/test-parameters.json

```diff
@@ -2,29 +2,32 @@
     "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "CosmosAccessKey": {
+            "value": "this-is-not-a-real-key=="
+        },
+        "CosmosDbCollectionName": {
+            "value": "helppages"
+        },
+        "CosmosDbName": {
+            "value": "dfc-dev-app-sharedresources-cdb"
+        },
+        "CosmosDbResourceGroup": {
+            "value": "dfc-dev-app-sharedresources-rg"
+        },
         "Environment": {
             "value": "FOO"
         },
+        "HelpAppDatabaseNeedsCreation": {
+            "value": true
```

Comment:
You'll need to set this to false on the second deploy of the arm templates.
Stupid, stupid limitation from Azure :(